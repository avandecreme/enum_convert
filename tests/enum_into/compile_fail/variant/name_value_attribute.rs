use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into = "Target::Bar"] // Should be #[enum_into(Target::Bar)]
    Foo,
}

enum Target {
    Bar,
}

fn main() {}
