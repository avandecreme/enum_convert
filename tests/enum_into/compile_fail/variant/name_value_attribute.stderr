error: expected a list of variants, for example #[enum_into(Target1::VariantA, Target2::VariantB)].
       If there is only one target enum and the variant names are identical between source and target, #[enum_into] can be omitted.
 --> tests/enum_into/compile_fail/variant/name_value_attribute.rs:6:5
  |
6 |     #[enum_into = "Target::Bar"] // Should be #[enum_into(Target::Bar)]
  |     ^
